bin/mantle-dynamics-L0/1-1_understanding_the_parameter_file.ipynb

```diff
@@ -0,0 +1 @@
this file, we only use global refinement as:\n","\n","```\n","subsection Mesh refinement\n","    set Initial global refinement = 3\n","    set Initial adaptive refinement = 0\n","    set Time steps between mesh refinement = 0\n","end\n","```\n","\n","### Material Model\n","This section describes the material properties of the model, such as density, viscosity, thermal conductivity etc. ASPECT provides various built in material models from whcih the user can choose. In this file, `simpler` model is used and, as specified here, has constant value for all material properties, except density which depends on temperature as :\n","\n","$\\rho (p, T) = \\rho_0 (1 - \\alpha (T - T_0))$,\n","\n","where, $rho_0$, $\\alpha$, $T$, represent the reference density, thermal expansivity, and temperature, respectively. The variations in density of the material lead to the resultant buoyancy forces needed to drive the convective flow.\n","\n","More details are available in the ASPECT manual on the formulation of the material model.\n","\n","```\n","subsection Material model\n","  set Model name = simpler\n","\n","  subsection Simpler model\n","    set Reference density             = 3300\n","    set Reference specific heat       = 1250\n","    set Reference temperature         = 293.\n","    set Thermal conductivity          = 4.7\n","    set Thermal expansion coefficient = 2e-5\n","    set Viscosity                     = 5.10452e24\n","  end\n","\n","end\n","```\n","\n","### Postprocessing\n","The postprocessing section of the input file tells ASPECT how to analyze the simulation data. The section below specifies heat flow and other statistics that are output with the solution. In this case, graphical output is generated every $10^7$ years.\n","\n","```\n","subsection Postprocess\n","    set List of postprocessors = velocity statistics, temperature statistics, heat flux statistics, visualization, basic statistics\n","    subsection Visualization\n","        set Time between graphical output = 1e7\n","        set Output format = vtu\n","        set List of output variables = material properties\n","    end\n","end\n","```"]},{"cell_type":"markdown","id":"c4e83e66","metadata":{"id":"c4e83e66"},"source":[" &nbsp;<div style=\"text-align: right\">  \n","    &rarr; <b>NEXT: [Convection in a box model](./2_convection_in_box_model.ipynb) </b> <a href=\"\"></a> &nbsp;&nbsp;\n","     <img src=\"../assets/education-gem-notebooks_icon.png\" alt=\"icon\"  style=\"width:4%\">\n","  </div>"]},{"cell_type":"code","source":[],"metadata":{"id":"kMhWb49oyboP"},"id":"kMhWb49oyboP","execution_count":null,"outputs":[]}],"metadata":{"kernelspec":{"display_name":"Python 3 (ipykernel)","language":"python","name":"python3"},"language_info":{"codemirror_mode":{"name":"ipython","version":3},"file_extension":".py","mimetype":"text/x-python","name":"python","nbconvert_exporter":"python","pygments_lexer":"ipython3","version":"3.11.5"},"colab":{"provenance":[]}},"nbformat":4,"nbformat_minor":5}
```

Comment:
<p>The next section is plate tecttonics and not Convection in a box model.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>